src/pages/Home/index.jsx

```diff
@@ -50,6 +55,35 @@ const Home = () => {
     }
   };
 
+  const fetchSketches = async () => {
+    try {
+      const response = await fetch(
+        `http://localhost:3000/sketches?per_page=3&page=${currentPage}`,
+      );
+      const sketchesData = await response.json();
+
+      if (sketchesData && sketchesData.sketchesUrl) {
+        setTotalPages(sketchesData.sketchesUrl.totalPages);
+        const sk = sketchesData.sketchesUrl.list.map((item) => item.imageUrl);
+        setSketches(sk);
```

Comment:
```suggestion
        const sk = sketchesData.sketchesUrl.list.map((item) => item.imageUrl);

        setTotalPages(sketchesData.sketchesUrl.totalPages);
        setSketches(sk);
```
변수명을 수정해주세요!
개행도 해주시는게 좋을 것 같아요